### DISCUSSION TITLE
Odroid support

### DISCUSSION THREAD

**Discussion by mattydebie** [tesla-android/issue-tracker#202] [Q&A] [answered]
Since the raspberry pi shortage around the world, is tesla-android also supported on the odroid family?

**Comment by mattydebie** [comment, marked as answer]
sorry, looked for odroid in the search, but apparently this question has been asked here: https://github.com/tesla-android/issue-tracker/discussions/78

**Comment by mikegapinski** [comment]
No, it uses APIs available only on the Pi + Android builds are not universal. I do have the Odroid C4 board, so who knows what the future holds ;)

**Reply by mattydebie** [reply]
Was just thinking about ordering the C4!
Maybe I should, and I could support you in adding support for the odroid

**Reply by mikegapinski** [reply]
My initial impressions of the C4 were like: meh. For the price it sucks. + the availability of Pi is improving
